plugins/GSdx/res/glsl/tfx_fs.glsl

```diff
@@ -28,7 +28,8 @@
 
 in SHADER
 {
-    vec4 t;
+    vec4 t_float;
+    vec4 t_int;
     vec4 c;
     flat vec4 fc;
 } PSin;
```

Comment:
The `#define PSin_t (PSin.t)` line a few lines below should probably be deleted.